src/screens/Booking/Booking.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
+
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import {RouteProp, useRoute} from '@react-navigation/native';
+import {FlightData} from '@components';
+import Baggage from './components/Baggage/Baggage';
+import Form from './components/PassengerForm/Form';
+import styles from './Booking.styles';
+import {RootStackParamsList} from '@navigation/types';
+
+const Booking = () => {
+  const route = useRoute<RouteProp<RootStackParamsList, 'BookingScreen'>>();
+  const [state, setState] = React.useState({
```

Comment:
why one state should keep 2 values inside if they have 2 different callbacks?
Just define 2 state: one for `cabin` value, second for `checked` value.